content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -217,20 +218,13 @@ export class CollapsibleSection extends React.PureComponent {
               <span
                 className="click-target"
                 role="button"
+                label={collapsed ? "Expand Section" : "Collapse Section"}
```

Comment:
should this be aria-label? Strings here should be coming from ftl file

Reply:
I'm not sure if it should be aria-label; I made it `label` based on @MarcoZehe 's review [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1547337) - but I'll try it and test the behaviour. I don't think it could hurt to have it be aria-label!

Reply:
Second the part on changing hard-coding strings to fluent. Looks like you'll have to create a new fluent ID since using any of the current ones will override/delete all your children with the fluent string.
https://github.com/mozilla/activity-stream/blob/8dccdc7c295bc583fb76d10e399d2f438a502d42/locales-src/newtab.ftl#L123-L125

New string with property assignment instead like
```
newtab-section-collapse-section-label =
    .aria-label = Collapse Section
```

Reply:
I was just looking for that, thanks!

Reply:
Reading the feedback above, should we add "Expand Section" or "Collapse Section " label to the arrow button only. My only concern is will this change make all section title read as expand or collapse section.
I am ok with this change if we can verify in screen reader or with @MarcoZehe that we are correctly reading titles of section e.g. TopSites, Recommended by Pocket ...

Reply:
Adding the label to the arrow button makes the final heading title/label "Top Sites Collapse Section", for example - I don't know if that's exactly what we want, but it does cover all the information at least.

Reply:
That should work, do we need title for that or should aria-label suffice? With title when mouse over alt text shows as 'Expand Section' not sure if thats desired.

Reply:
I'll leave it as just aria-label for now, we can always tweak it further.